client/sites-dashboard-v2/sites-dashboard.tsx

```diff
@@ -231,6 +233,12 @@ const SitesDashboardV2 = ( {
 					<LayoutTop withNavigation={ false }>
 						<LayoutHeader>
 							{ ! isNarrowView && <Title>{ translate( 'Sites' ) }</Title> }
+							<Subtitle>
+								{ translate( 'Manage all your sites' ) }
```

Comment:
I want to change this to "Manage all **of** your sites," but I see Domains is the same, so that's probably a follow-up.